src/io/http/src/test/scala/ContinuousHTTPSuite.scala

```diff
@@ -175,6 +180,29 @@ class ContinuousHTTPSuite extends TestBase with HTTPTestUtils {
     val meanLatency = mean(latencies)
     val stdLatency = stddev(latencies, meanLatency)
     println(s"Latency = $meanLatency +/- $stdLatency")
+    assert(meanLatency < 5)
+
+    println(HTTPSourceStateHolder.serviceInfoJson(apiName))
+    println("stopping server")
+    server.stop()
```

Comment:
is there any way for you to use a using (...) { } like structure to make sure exceptions are properly handled (e.g. during unit tests the servers are properly shutdown)